translations/src/config.js

```diff
@@ -34,8 +34,8 @@ class Config {
     tr: { rtl: false },
     pl: { rtl: false },
     ti: { rtl: false },
-    ku: {
-      rtl: false,
+    ckb: {
```

Comment:
Does it make sense to use ckb-Latn. ckb also uses a non latin alphabet in our translations.

Reply:
It may make sense for Sorani.
From https://de.wikipedia.org/wiki/Sorani:
> Zur Schreibung des Sorani wird meist die Persische Schrift mit Sonderzeichen verwendet, zunehmend aber auch das kurdisch-lateinische Alphabet.

Lets discuss this in the App Call later.

Reply:
does that mean we already have ckb translations? Where could I find them? And of course if theire implementing the latin alphabet it should be ckb-Latn. Even though in our wiki we explicitly write ckb with persian alphabet.

Reply:
I don't think we have ckb translations and I also don't think we will have ckb-Latn. But how should we know? :D